dsc/assertion.dsc.resource.json

```diff
@@ -1,5 +1,5 @@
 {
-  "manifestVersion": "1.0",
+  "$schema": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/bundled/resource/manifest.json",
```

Comment:
I don't think it matters at this point, but keep in mind that GitHub is going to server this as "text/plain" and at some point we will probably want/need to have it served as "application/schema+json". There's a short discussion about this in the JSON Schemas repo: https://github.com/orgs/json-schema-org/discussions/29

Reply:
Long term, we need to decide if we want to publish the schemas to https://github.com/Microsoft/json-schemas or keep it in the repo.  There's pros/cons for both.  Not a blocker for this PR.